website/views.py

```diff
@@ -261,7 +265,7 @@ def watched_logs_get(**kwargs):
         ))
 
     total = sum(1 for x in user.get_recent_log_ids())
-    paginated_logs, pages = paginate(user.get_recent_log_ids(), total, page, size)
+    paginated_logs, pages, page = paginate(user.get_recent_log_ids(), total, page, size)
```

Comment:
We shouldn't overwrite `page` here, as it's already been defined in this scope.

I see that it is passed around to allow dynamically changing the page returned when the user requests and out-of-range value, but as I mentioned above, let's not do that.